wordpress-mobile/WordPress-Android
# Issue/7709 whats next sites

### Fix
Add a ***What's Next*** list item to the ***Sites*** tab as described in https://github.com/wordpress-mobile/WordPress-Android/issues/7709.  This ***does not*** include triggering ***What's Next*** prompts when the user visits an associated screen as originally planned.  Those triggers are dependent upon the checklist progress determined in a subsequent issue.  The only trigger included is the initial ***What's Next*** dialog shown when creating a new site.

There are multiple differences in the interface since the original design.  See the descriptions and screenshots below for details.
- What's Next was changed to Quick Start
- Prompt for help changed from [`WPDialogSnackbar`](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/whats-next/WordPress/src/main/java/org/wordpress/android/widgets/WPDialogSnackbar.java) to [`PromoDialog`](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/whats-next/WordPress/src/main/java/org/wordpress/android/ui/posts/PromoDialog.kt)
- Progress (i.e. 1/7) text color and size updated to match style of other views (i.e. Plan)
- Total number shown in progress is 7 to match the number of tasks
- Never button added to prompt asking for help
- Copy updated after Editorial review

![sites_permission](https://user-images.githubusercontent.com/3827611/40255100-3099b3f4-5aa3-11e8-9065-734dcf1b5e91.png)

![sites_progress](https://user-images.githubusercontent.com/3827611/40255170-73312d46-5aa3-11e8-849e-bc93e7441a76.png)

![sites_completed](https://user-images.githubusercontent.com/3827611/40255172-74727ac0-5aa3-11e8-8786-a84a71a668dc.png)

#### Note
The ***Quick Start*** view is invisible.  It will become visible in a subsequent issue.

### Test
1. Go to ***Sites*** tab.
2. Tap ***Switch Site*** button.
3. Tap ***+*** action.
4. Tap ***Create WordPress.com site*** option.
5. Go through creating site.
6. Tap ***Configure*** button.
7. Notice ***Quick Start*** dialog is shown.
